make test does not find gocov and gocov-xml.

Maybe `make test` should install missing deps.

gocov convert ./unit_coverage.cov | gocov-xml > ./coverage.xml
/bin/bash: gocov-xml: command not found
/bin/bash: gocov: command not found
make: *** [test] Error 127

--- a/.github/workflows/build-go.yaml
+++ b/.github/workflows/build-go.yaml
@@ -35,8 +35,6 @@ jobs:
           go install go.aporeto.io/regolithe/cmd/rego@master
           go install github.com/securego/gosec/cmd/gosec@latest
           go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
-          go install github.com/axw/gocov/gocov@latest
-          go install github.com/AlekSi/gocov-xml@latest
 
       - name: build
         run: |

--- a/Makefile
+++ b/Makefile
@@ -32,8 +32,8 @@ lint:
 		./...
 
 testdeps:
-	go install github.com/axw/gocov/gocov@master
-	go install github.com/AlekSi/gocov-xml@master
+	go install github.com/axw/gocov/gocov@latest
+	go install github.com/AlekSi/gocov-xml@latest
 
 test: testdeps
 	go test ./... -race -cover -covermode=atomic -coverprofile=unit_coverage.cov

--- a/Makefile
+++ b/Makefile
@@ -31,7 +31,11 @@ lint:
 		--enable=nilerr \
 		./...
 
-test:
+testdeps:
+	go install github.com/axw/gocov/gocov@master
+	go install github.com/AlekSi/gocov-xml@master
+
+test: testdeps
 	go test ./... -race -cover -covermode=atomic -coverprofile=unit_coverage.cov
 	gocov convert ./unit_coverage.cov | gocov-xml > ./coverage.xml
 

